include/preprocess/crhmc/opts.h

```diff
@@ -0,0 +1,38 @@
+// VolEsti (volume computation and sampling library)
```

Comment:
The parameters (I will put the other constants as well), I am thinking that it can have the parameters of all the algorithm (also of the ODE and Sampler part).

Reply:
thanks!